### PR TITLE
chore: minor changes to thorchain swapper tx status during swaps

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Investigation of #7371 led to some improvements to copy and a fix to swap status logic. Huzzah.

## Issue (if applicable)

Stemmed from investigation of #7371

## Risk

> High Risk PRs Require 2 approvals

Low risk.

> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

Test thorchain swaps complete as expected with better copy.

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

## Screenshots (if applicable)

image

image

### CODE REVIEW

File: packages/swapper/src/swappers/ThorchainSwapper/endpoints.ts [hunk 1]
```diff
@@ -373,7 +373,7 @@ export const thorchainApi: SwapperApi = {
       }
 
       const latestOutTx = data.out_txs?.[data.out_txs.length - 1]
-      const hasOutboundTx = latestOutTx?.chain !== 'THOR'
+      const hasOutboundTx = latestOutTx !== undefined && latestOutTx.chain !== 'THOR'
```
Comment (woodenfurniture) [inline packages/swapper/src/swappers/ThorchainSwapper/endpoints.ts:376]: So turns out, if there is no `latestOutTx` it means it doesn't actually have an outbound tx 😅

File: packages/swapper/src/swappers/ThorchainSwapper/utils/getLatestThorTxStatusMessage.ts [hunk 2]
```diff
@@ -61,15 +61,18 @@ export const getLatestThorTxStatusMessage = (
           }
         }
 
-        return { message: obj.pending ? 'Swap pending' : 'Swap complete', status: TxStatus.Pending }
+        return {
+          message: obj.pending ? 'Swap pending' : 'Swap complete, awaiting finalization',
```
Comment (0xApotheosis) [inline packages/swapper/src/swappers/ThorchainSwapper/utils/getLatestThorTxStatusMessage.ts:65]: Not for this PR at all, but it occurred to me whilst reading this that all of these probably need actual translations!

**Comment by woodenfurniture** [comment]
Closed in favor of #7831

**Comment by woodenfurniture** [comment]
Closed in favor of #7831

**Review by 0xApotheosis — APPROVED** [review]
gm

Small streaming swap with appropriate messages:

Screenshot 2024-09-30 at 13 33 43
Screenshot 2024-09-30 at 13 34 01
Screenshot 2024-09-30 at 13 34 24

**Comment by 0xApotheosis** [comment]
The ruggiest of rugs 😂

**Pull request closed without merging**